recipes/poppler/all/conanfile.py

```diff
@@ -132,125 +126,119 @@ def _minimum_compilers_version(self):
+        tc.variables["WITH_NSS3"] = self.options.with_nss
+        tc.variables["WITH_Cairo"] = self.options.with_cairo
+        tc.variables["ENABLE_GLIB"] = self.options.get_safe("with_glib", False)
+        tc.variables["ENABLE_GOBJECT_INTROSPECTION"] = self.options.get_safe("with_gobject_introspection", False)
+        tc.variables["WITH_Iconv"] = self.options.get_safe("with_libiconv")
+        tc.variables["ENABLE_ZLIB"] = self.options.with_zlib
+        tc.variables["ENABLE_LIBOPENJPEG"] = "openjpeg2" if self.options.with_openjpeg else "none"
+        tc.variables["ENABLE_CMS"] = "lcms2" if self.options.with_lcms else "none"
+        tc.variables["ENABLE_LIBCURL"] = self.options.with_libcurl
+
+        tc.variables["POPPLER_DATADIR"] = self.deps_user_info["poppler-data"].datadir.replace("\\", "/")
+        tc.variables["FONT_CONFIGURATION"] = self.options.fontconfiguration
+        tc.variables["BUILD_CPP_TESTS"] = False
+        tc.variables["ENABLE_GTK_DOC"] = False
+        tc.variables["ENABLE_QT5"] = self.options.with_qt and Version(self.deps_cpp_info["qt"].version).major == "5"
+        tc.variables["ENABLE_QT6"] = self.options.with_qt and Version(self.deps_cpp_info["qt"].version).major == "6"
+
+        tc.variables["ENABLE_DCTDECODER"] = self._dct_decoder
+        tc.variables["USE_FLOAT"] = self.options.float
+        tc.variables["RUN_GPERF_IF_PRESENT"] = False
         if self.settings.os == "Windows":
-            self._cmake.definitions["ENABLE_RELOCATABLE"] = self.options.shared
-        self._cmake.definitions["EXTRA_WARN"] = False
+            tc.variables["ENABLE_RELOCATABLE"] = self.options.shared
+        tc.variables["EXTRA_WARN"] = False
 
         # Workaround for cross-build to at least iOS/tvOS/watchOS,
         # when dependencies are found with find_path() and find_library()
-        if tools.cross_building(self):
-            self._cmake.definitions["CMAKE_FIND_ROOT_PATH_MODE_INCLUDE"] = "BOTH"
-            self._cmake.definitions["CMAKE_FIND_ROOT_PATH_MODE_LIBRARY"] = "BOTH"
-
-        self._cmake.configure(build_folder=self._build_subfolder)
-        return self._cmake
-
-    def _patch_sources(self):
-        for patchdata in self.conan_data["patches"][self.version]:
-            tools.patch(**patchdata)
-        if tools.Version(self.version) < "21.07.0" and not self.options.shared:
-            poppler_global = os.path.join(self._source_subfolder, "cpp", "poppler-global.h")
-            tools.replace_in_file(poppler_global, "__declspec(dllimport)", "")
-            tools.replace_in_file(poppler_global, "__declspec(dllexport)", "")
-        tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"),
-                              "FREETYPE_INCLUDE_DIRS",
-                              "Freetype_INCLUDE_DIRS")
+        if cross_building(self):
+            tc.variables["CMAKE_FIND_ROOT_PATH_MODE_INCLUDE"] = "BOTH"
+            tc.variables["CMAKE_FIND_ROOT_PATH_MODE_LIBRARY"] = "BOTH"
```

Comment:
@jcar87 I'd love to have your expertise one this is out of my wheel house ❤️‍🔥

Reply:
For this one I'm not 100% sure if it's required since I don't do cross compilation myself, so this is just copied from the old version of the recipe.